### ISSUE TITLE
iNaturalist data - Recorder details

### ISSUE THREAD

**Issue by MNSmith** [BiologicalRecordsCentre/iRecord#582]
The iNaturalist import seems to be importing the "Username" from iNats into iRecord, and a number of verifiers have commented that they are not happy with this and would prefer a "proper" name.  I'm ok with this providing that at present the record comes through with a photo.  However, it has been noted that when setting up an iNats account you do actually input your full details.  While I would prefer a full name to come across and that the link might be tweaked to accomplish this, I understand that this may not be possible due to the T&Cs at the iNats end with regards export of full names.  Some thoughts on this might be helpful.

Aprapos to this, if you look at the "Experience" tag of an imported iNats record, the message displayed is "Recorder was not logged in so experience cannot be loaded.".  With iNats imports the Recorder is never going to be logged in.  Would it be possible to begin generating Recorder statistics for iNats imports based on verification decisions?  This would be useful, particularly if we can build up a recorder experience profile such that verifiers do feel able to accept records with no photos based on "Recorder experience".

**Comment by johnvanbreda** [comment]
Full name details are optional in iNat so we use the login name if the iNat API doesn't give us a full name. There is probably not much we can do about this. 
We might be able to extend the import to capture a unique user ID in all cases, then use this to build a user experience tab. It'll need @kitenetter to agree it's a priority though as it's a few hours work. Note though now you can click through to the iNat record so it might be slightly easier to check up on unknown recorders.

**Comment by kitenetter** [comment]
I've also seen one or two examples on iNat where users have put their real name in the "username" category and a nickname in the "real name" category, so it is hard to parse all the potential variation.

We could look at importing a unique user ID but I don't think that can be a priority at the moment, so hopefully the click-through option will act as a work-around for now.

**Comment by MNSmith** [comment]
With iNats records, iRecord displays the following text:

"View details of this record in iNaturalist using the link above. Although there are no media files linked to the imported record, this may be because the source record's images were not licensed so could not be imported. If so then they may be viewed in iNaturalist."

There is no link visible or active that I can see, I can't go to iNats directly from iRecord.

**Comment by johnvanbreda** [comment]
@MNSmith do you not see the extra record details button in the row of tool buttons (alongside the existing record details button for iRecord), e.g. 
![Screenshot 2019-04-29 at 12 20 38](https://user-images.githubusercontent.com/2392594/56893077-67688180-6a79-11e9-8b72-3a8997938eba.png)

**Comment by MNSmith** [comment]
Thanks - was looking for a link in the Record Details pane, not an action button.

**Comment by johnvanbreda** [comment]
No problem. It seemed more consistent to me to put the record details buttons together for iNat and iRecord.

**Comment by UrkStarkadder** [comment]
I can cope with usernames in lieu of a recorder but far more important IMHO is that we don't lose the names of the folks who actually do the identifying. I know of one person who has given hundreds of hours to iNat who would be very irked indeed if he discovered that his name wasn't going through to any subsequent user

**Comment by kitenetter** [comment]
Thanks for the comment @UrkStarkadder. We don't have any scope for further enhancements to the import process at the moment but will review later in the year.
